garageScript/c0d3-app
# Include Discord user info in the session query

Related to #2345 

### Description

This PR adds the `discordUserInfo`, `discordUsername`, and `discordAvatarUrl` to the `getSession` query. These will be used in a subsequent PR to cache these values.

### Testing

1. Connect to discord
2. Go to GraphQL playground `/api/graphql`
3. Set `request.credentials` to `include` in the playground settings
4. Run

```gql
query {
  session {
    user {
      discordUsername
      discordUserId
      discordAvatarUrl
    }
  }
}
```

5. Make sure it returns the data correctly
